### PR TITLE
{Tidy-up}: {Cleaned up some code}

### PR DESCRIPTION
**Pull request by PythonCoderAS**
Clean up

## Checklist

- [x] Your branch is up-to-date with the base branch
- [ ] You've included at least one test if this is a new feature
- [x] All tests are passing

### CODE REVIEW

**Comment by ManrajGrover** [comment]
@PokestarFan Thanks for this PR. 💯 I'll review it as soon as possible.

File: halo/halo.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
+#!usr/bin/env python
```
Comment (ManrajGrover) [inline halo/halo.py]: @PokestarFan Any particular reason why you have added this shebang?

**Comment by ManrajGrover** [comment]
@PokestarFan Are you still on this PR? I plan to merge this after confirmation.

**Comment by PythonCoderAS** [comment]
Hello it's been 3 years! I totally forgot about this PR and only found it when I accidentally clicked into https://github.com/pulls! The PR is out of date so I'll close it now.

**Pull request closed without merging**